bot/exts/easter/bunny_name_generator.py

```diff
@@ -18,15 +18,15 @@ class BunnyNameGenerator(commands.Cog):
     """Generate a random bunny name, or bunnify your Discord username!"""
 
     @staticmethod
-    def find_separators(displayname: str) -> Union[List[str], None]:
+    def find_separators(displayname: str) -> Optional[list[str]]:
         """Check if Discord name contains spaces so we can bunnify an individual word in the name."""
         new_name = re.split(r"[_.\s]", displayname)
         if displayname not in new_name:
             return new_name
         return None
 
     @staticmethod
-    def find_vowels(displayname: str) -> str:
+    def find_vowels(displayname: str) -> Optional[str]:
```

Comment:
You can add `return None` at the end of this method.

Reply:
There is no need for this. It's either that the `return new_name` is hit within the for loop, or the function returns the default value of `None`.

Reply:
I don't get the convention because sometimes `None` is returned, sometimes not 🤔

Reply:
Which convention? The Optional annotation?